tests/integration.test.js

```diff
@@ -325,7 +325,7 @@ tap.test('views integration', async t => {
   t.end()
 })
 
-tap.test('encryption integration', async t => {
+tap.skip('encryption integration', async t => {
```

Comment:
Could clarify the reason why we're skipping this test?

Reply:
Oh, I forget it.